client/App.tsx

```diff
@@ -1,7 +1,83 @@
-import React from "react";
+import React, { useState } from "react";
+
+import { useRegisterMutation } from "./features/api";
+
+// TODO: RegisterForm component only added for testing in this branch
+const RegisterForm = () => {
+
+  const [register, { isLoading, isError, data }] = useRegisterMutation();
+
+  const [email, setEmail] = useState("");
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+
+  if (data) {
+    console.log(data);
+  }
```

Comment:
I wouldn't recommend this - i'd recommend instead using `useSelector` in your App.tsx to get the token and logging from there, to confirm that the auth slice also works